### ISSUE TITLE
[ADD RULE] WCAG 2.2: 2.5.8 Target size

### ISSUE THREAD

**Issue by j-mendez** [a11ywatch/kayle#22]
Part of adding the new rules from 2.2.

The 2.5.8 needs to be implemented in fast_axecore and fast_htmlcs.

We have a base rule that iterates the nodes top level for htmlcs. The htmlcs portion should include the logic to check for the elements next to each other to see if the rule fails.

fast_axecore, we do not have as nice of a setup for all top rules. We may need to just add a new rule to go through all nodes.

Being cautious of performance is critical for this rule since it touches all nodes.
